package.json

```diff
@@ -1,26 +1,26 @@
 {
   "dependencies": {
-    "@fortawesome/fontawesome-free": "^5.15.2",
-    "bootstrap": "^4.6.1",
+    "@fortawesome/fontawesome-free": "^6.4.2",
+    "bootstrap": "^5.3.1",
```

Comment:
<!-- Issues represent user-facing problems. If possible, it's great to follow this kind of comment with a suggestion.  -->
**🔨 issue:** ‏sorry is not possible to update all the frontend frameworks without updating also the templates  and adapt the layouts. everything is designed around bs4, so the theme switcher and other components will break without proper adaptiation.